electron/logger/logger.ts

```diff
@@ -100,10 +135,23 @@ export function logInfo(
   prefix: LogPrefix = LogPrefix.General,
   skipLogToFile = false
 ) {
-  const prefixString = prefix !== LogPrefix.General ? `[${prefix}]: ` : ''
-  const extendText = `INFO: ${prefixString}${convertToStringArray(text).join(
-    ' '
-  )}`
+  // time
+  const ts = new Date()
+  const timeString = `(${[
```

Comment:
if this is repeated so much, perhaps create another function that returns this result.